clang/lib/DPCT/Diagnostics.inc

```diff
@@ -241,3 +241,5 @@ DEF_WARNING(TEX_FETCH, 1112, "If the filter mode is set to 'linear', the behavio
 DEF_COMMENT(TEX_FETCH, 1112, "If the filter mode is set to 'linear', the behavior of \"read\" may be different from \"tex1Dfetch\". You may need to adjust the code.")
 DEF_WARNING(ONE_DIMENSION_KERNEL_BARRIER, 1113, "Consider replacing sycl::nd_item::barrier(sycl::access::fence_space::local_space) with sycl::nd_item::barrier() if function \"%0\" is called in a multidimensional kernel.")
 DEF_COMMENT(ONE_DIMENSION_KERNEL_BARRIER, 1113, "Consider replacing sycl::nd_item::barrier(sycl::access::fence_space::local_space) with sycl::nd_item::barrier() if function \"{0}\" is called in a multidimensional kernel.")
+DEF_WARNING(WAIT_REMOVE, 1114, "There is no synchronize operation append after memcpy. This is based on the assumption that the original cudaMemcpy operations are asynchronous when data is transferred to the device and the memory involved is neither Unified Memory nor pinned memory. If this assumption doesn't hold, please append .wait() after memcpy for correct synchronization.")
```

Comment:
In https://docs.nvidia.com/cuda/cuda-runtime-api/api-sync-behavior.html#api-sync-behavior__memcpy-sync , it only mentioned the terminologies "pageable memory" and "pinned memory" but no "Unified Memory".
Is it confusing that we mention "Unified Memory" here?